test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -140,6 +140,47 @@ public void TestWillCreateTrxLogger()
             }
         }
 
+        [Fact]
```

Comment:
This test is testing both the features : a.) newly added log file name. b.) custom RunSettingsOptions. 
`--logger \"trx;logfilename=custom.trx\" -- RunConfiguration.ResultsDirectory=" + trxLoggerDirectory`